app/templating/summary/question.py

```diff
@@ -0,0 +1,76 @@
+from app.questionnaire.navigator import evaluate_rule
+from app.templating.summary.answer import Answer
+
+
+class Question:
+
+    def __init__(self, block_id, question_schema, answers):
+        self.id = question_schema['id']
+        self.type = question_schema['type']
+        self.skip_condition = question_schema.get('skip_condition')
+        answer_schema = question_schema['answers']
+        self.title = question_schema['title'] or answer_schema[0]['label']
+        self.answers = Question._build_answers(block_id, question_schema, answer_schema, answers)
+
+    def is_skipped(self, answers):
+        if self.skip_condition is not None:
+            answer = answers[self.skip_condition['when']['id']]
+            return evaluate_rule(self.skip_condition, answer)
+
+        return False
+
+    @staticmethod
+    def _build_answers(block_id, question_schema, answer_schema, answers):
```

Comment:
Does it make sense that these methods are instance methods and the class holds the arguments when initialised, so we don't have to keep passing them in?